src/Text.php

```diff
@@ -6,6 +6,18 @@
 
 namespace karmabunny\kb;
 
+use Exception;
+
+enum TextMaskTypes {
+    case FirstName;
+    case LastName;
+    case FirstLastName;
+    case Email;
+    case Phone;
+    case Spaces;
+}
```

Comment:
This library is currently targeting php 7.2 which doesn't have enums.

I wouldn't say no to a major version that only supports the 8.x series so we can use them. We'd end up having to backport bug fixes for a lot of things. It might be a nice clean slate though, we'd get lots of typey safety stuff from it.

That all said, knowing what project this code is targeted for - it's also still only on php 7.2 (one of the last!)

What's also problematic is that this enum is not discoverable by PSR-4 convention. It needs to live it it's own file TextMaskTypes. The only reason it works in the test suite is because the Text class is autoloaded first which pulls this whole file.